source/openapi-admin-v3.yaml

```diff
@@ -1670,6 +1670,42 @@ paths:
         "204":
           description: Deleted
 
+  "/groups/{groupId}/apps/{appId}/users/{userId}/reset_email":
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+      - "$ref": "#/components/parameters/UserId"
+    patch:
+      tags:
+        - users
+      operationId: adminResetUserEmail
+      summary: Change a user's email
+      description: Change the email address for an [email/password user](https://www.mongodb.com/docs/atlas/app-services/authentication/email-password/) by ID.
+      requestBody:
+        required: true
+        description: The new email address for the user.
+        content:
+          application/json:
+            schema:
+              properties:
+                email:
+                  type: string
+                  description: The new email for the user.
+              required:
+                - email
+            example: { "email": "new_email@example.com" }
+      responses:
+        "204":
+          description: No Content
+        "400":
+          description: |-
+            - Payload is formatted incorrectly.
+            - The email in the payload is not a string.
+            - The email in payload has 0 or greater than 128 characters.
+            - The user does not have a local-userpass identity.
```

Comment:
Huh, I'm a urprised this isn't a 404.

Reply:
I actually meant to ask you - these are listed in the spec as 400s, but I didn't actually get this wording for any of these conditions. I thought it was useful to list these as reasons you could get a 400, but I was wondering if this "description" field is meant to be only the string text that accompanies the status (which was something like malformed payload, IIRC), or if I can use it like this to list the potential causes for the status.